var/spack/repos/builtin/packages/hermes/package.py

```diff
@@ -11,38 +11,136 @@ class Hermes(CMakePackage):
     I/O buffering system that aims to significantly accelerate I/O performance.
     """
 
-    homepage = "http://www.cs.iit.edu/~scs/assets/projects/Hermes/Hermes.html"
+    homepage = "https://grc.iit.edu/research/projects/hermes"
     git = "https://github.com/HDFGroup/hermes.git"
 
-    maintainers("hyoklee")
+    maintainers("lukemartinlogan", "hyoklee")
 
-    license("GPL-2.0-only")
+    version("master", branch="master", submodules=True)
 
-    version("master", branch="master")
+    version(
+        "1.2.1",
+        url="https://github.com/HDFGroup/hermes/archive/refs/tags/v1.2.1.tar.gz",
+        sha256="d60ee5d6856dc1a1f389fb08f61252cc7736d1c38d3049043749640897fe3b6d",
+    )
+    version(
+        "1.2.0",
+        url="https://github.com/HDFGroup/hermes/archive/refs/tags/v1.2.0.tar.gz",
+        sha256="280379f393695462279c1af11371995d134f33998596812bd960d6f44a86c339",
+    )
+    version(
+        "1.1.0",
+        url="https://github.com/HDFGroup/hermes/archive/refs/tags/v1.1.0.tar.gz",
+        sha256="022df20d9e394754f6126dfcedd845afde879fcb2738d763f53657e59e058c1e",
+    )
+    version(
+        "1.0.5-beta",
+        url="https://github.com/HDFGroup/hermes/archive/refs/tags/v1.0.5-beta.tar.gz",
+        sha256="1f3ba51a8beda4bc1314d6541b800de1525f5e233a6f498fcde6dc43562ddcb7",
+    )
+    version(
+        "1.0.0-beta",
+        url="https://github.com/HDFGroup/hermes/archive/refs/tags/v1.0.0-beta.tar.gz",
+        sha256="301084cced32aa00532ab4bebd638c31b0512c881ffab20bf5da4b7739defac2",
+    )
+    version(
+        "0.9.9-beta",
+        url="https://github.com/HDFGroup/hermes/archive/refs/tags/v0.9.9-beta.tar.gz",
+        sha256="d2e0025a9bd7a3f05d3ab608c727ed15d86ed30cf582549fe996875daf6cb649",
+    )
+    version(
+        "0.9.8-beta",
+        url="https://github.com/HDFGroup/hermes/archive/refs/tags/v0.9.8-beta.tar.gz",
+        sha256="68e9a977c25c53dcab7d7f6ef0df96b2ba4a09a06aa7c4a490c67faa2a78f077",
+    )
+    version(
+        "0.9.5-beta",
+        url="https://github.com/HDFGroup/hermes/archive/refs/tags/v0.9.5-beta.tar.gz",
+        sha256="f48d15591a6596e8e54897362ec2591bc71e5de92933651f4768145e256336ca",
+    )
```

Comment:
Just as a question (just curious): Do you need all those versions?

If there is no requirement to have the checksums of all these versions, I would not add them, because versions accumulate anyway, and once you have versions listed, you'd have to deprecate them before removing them again to limit the size of the recipe.
```suggestion
```